module/common/Logging.h

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file Logging.h
+ * @brief 動作確認に用いるprintf()関数を所持するクラス
+ * @author sap2368
+ */
+#ifndef LOGGING_H
+#define LOGGING_H
+
+#include <stdio.h>
+
+class Logging {
+ public:
+  Logging();
+
+  //入力された文字列をLogとしてターミナルに表示
+  void writeLogs(const char* logs);
```

Comment:
コメント書いてとしか言わなかったけど，他のファイルみたいにdoxygen形式のコメントで書いてほしい．他の関数も同様で

あと，他は変更したけどここは複数形（logs）なのはなんで？修正忘れ？

```suggestion
  /**
  ** 入力された文字列をLogとしてターミナルに表示
  ** @param logs
  **/
  void writeLogs(const char* logs);
```